### PR TITLE
[1250] add new course cta

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Add new course

### Changes proposed in this pull request
"Add new course" CTA to `courses#index`

### Guidance to review
# After
localhost_3000_organisations_2AT_courses

### CODE REVIEW

File: app/views/courses/index.html.erb [hunk 1]
```diff
@@ -14,12 +14,15 @@
...
 <p class="govuk-body">Use this section to:</p>
 
-<ul class="govuk-list govuk-list--bullet">
+<ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-7">
   <li>write about each course</li>
   <li>preview and publish courses</li>
   <li>copy content between courses</li>
 </ul>
 
+<%= link_to "Add a new course", "https://goo.gl/forms/kdNksCuwcz7TVtNF2", class: "govuk-button govuk-!-margin-bottom-2", rel: "noopener noreferrer", target: :blank %>
```
Comment (defong) [inline app/views/courses/index.html.erb]: Should it be in settings?

File: app/views/courses/index.html.erb [hunk 2]
```diff
@@ -30,3 +33,6 @@
 
   <%= render partial: 'course_table', locals: { courses: courses } %>
 <% end %>
+
+<%= link_to "Add a new course", "https://goo.gl/forms/kdNksCuwcz7TVtNF2", class: "govuk-button govuk-!-margin-bottom-2", rel: "noopener noreferrer", target: :blank %>
```
Comment (defong) [inline app/views/courses/index.html.erb]: Missing tests?

File: app/views/courses/index.html.erb (hunk 1 above)
Comment (fofr) [inline app/views/courses/index.html.erb]: We need to link to two different forms based on the type of provider.

There's one form for Universities and SCITTs and another for School Direct providers.

Comment (fofr) [inline app/views/courses/index.html.erb]: We should only show the "Add a new course" link at the top of there's more than 10 courses.

Comment (fofr) [reply app/views/courses/index.html.erb]: Probably also needs to be wrapped in `@provider.opted_in?`?

**Comment by benilovj** [comment]
@dankmitchell I've added 449a92be5604085b14d41496840cebc161fc3a75 which pulls all the Google form URLs into `Settings`, so it's easier to see which one is which.

File: spec/helpers/provider_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+feature 'View helpers', type: :helper do
+  skip "#add_course_link" do
+    it "returns correct google form for accrediting providers" do
+      binding.pry;
```
Comment (defong) [inline spec/helpers/provider_helper_spec.rb]: Binding pry. You made my day!

File: spec/features/courses/index_spec.rb [hunk 4]
```diff
@@ -62,9 +61,17 @@
         expect(find('[data-qa="courses-table__course"]')).to have_content(course_3.attributes[:name])
       end
     end
+
+    scenario "it shows 'add a new course' link" do
```
Comment (defong) [inline spec/features/courses/index_spec.rb]: Maybe overkill but there can be 2 'add a new course' links

Comment (dankmitchell) [reply spec/features/courses/index_spec.rb]: Overkill.

File: spec/helpers/provider_helper_spec.rb (hunk 3 above)
Comment (dankmitchell) [reply spec/helpers/provider_helper_spec.rb]: haha was "wip"

**Comment by defong** [comment]
[Add New Training Courses Button](https://trello.com/c/70DB4WBq/1250-add-new-training-courses-button)

**Pull request merged**